src/renderer/components/+catalog/catalog-menu.tsx

```diff
@@ -43,7 +43,7 @@ function getCategories() {
 function getCategoryIcon(category: CatalogCategory) {
   if (!category.metadata?.icon) return null;
 
-  return category.metadata.icon.includes("<svg")
+  return category.metadata.icon.includes?.("<svg")
```

Comment:
why is this here? would it be better to just check if the `typeof` above is correct?

Reply:
Not sure that (adding `typeof` checks to `category.metadata?.icon`), will be the original intent of the code, there was no spec. But I guess both should be fine...